src/figuresClasses.ts

```diff
@@ -1,11 +1,79 @@
-export interface Figure {}
+interface Figure {
+  shape: `triangle` | `circle` | `rectangle`;
+  color: `red` | `green` | `blue`;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: 'triangle' = 'triangle';
```

Comment:
```suggestion
  shape: Shape = 'triangle';
```

Use type that you will create here